sphinx_design/dropdown.py

```diff
@@ -142,7 +142,11 @@ class DropdownHtmlTransform(SphinxPostTransform):
 
     def run(self):
         """Run the transform"""
-        for node in self.document.traverse(lambda node: is_component(node, "dropdown")):
+        # Can just use "findall" once docutils 0.18.1+ is required
+        meth = "findall" if hasattr(self.document, "findall") else "traverse"
+        for node in getattr(self.document, meth)(
```

Comment:
```suggestion
        for node in getattr(self.document, "findall", self.document.traverse)(
```

this is more sucinct no?

Reply:
If they do actually remove `.traverse` this new version breaks, hence the `hasattr` approach. But maybe by then you will  have moved to requiring a new enough version anyway...